CHANGELOG.md

```diff
@@ -6,9 +6,14 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 The following changes have been implemented but not released yet:
 
+### Bugfix
+
+- Export all types via Rollup.
+- Fix demo application.
+
 ### New features
 
--
+- Drop Node 12 support.
```

Comment:
This is also a breaking change, not a new feature.

Reply:
Marked as such.